server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/qualitygate/QualityGateServiceImplTest.java

```diff
@@ -25,7 +25,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.sonar.db.DbClient;
-import org.sonar.db.DbSession;
+import org.sonar.db.DbSessionImpl;
```

Comment:
Using `DbSessionImpl` is not required in this test. It can be replaced by `DbSession`.